README.md

```diff
@@ -1 +1,71 @@
 # Vehicle App C++ Template
+
+![Template CI Workflow](https://github.com/eclipse-velocitas/vehicle-app-cpp-template/actions/workflows/ci.yml/badge.svg)
+[![License: Apache](https://img.shields.io/badge/License-Apache-yellow.svg)](http://www.apache.org/licenses/LICENSE-2.0)
+
+The Vehicle App Template for C++ allows to create `Vehicle Apps` from the [Velocitas](https://github.com/eclipse-velocitas/velocitas-docs) development model in the C++ programming language.
+
+## Folder structure
+
+* 📁 `app` - base directory for a vehicle app
+    * 📁 `src` - source code of the vehicle app
+    * 📁 `tests` - tests for the vehicle app
+    * 📁 `vehicle_model` - vehicle model to be used by the vehicle app
```

Comment:
Is the vehicle model specific for this app (in case there are multiple) as shown here or is it part of the development environment (as in .vscode/vss.json)?

Reply:
A model can be shared between multiple apps but an app is tailored for exactly one model of the developers choosing at this point in time.